gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -89,10 +92,27 @@ def add_arguments(self, parser):
               'variants. This is useful when there are header fields in '
               'variants not defined in the header sections.'))
 
+  def validate(self, parsed_args):
+    input_pattern_re_match = re.match(
```

Comment:
actually, the input_pattern can be a local file as well. I actually don't think we need this check as the one below it would cover invalid paths as well.

Reply:
removed.